apps/src/appMain.js

```diff
@@ -52,7 +52,8 @@ module.exports = function(app, levels, options) {
   if (studioApp.isUsingBlockly()) {
     var blockInstallOptions = {
       skin: options.skin,
-      isK1: options.level && options.level.isK1
+      isK1: options.level && options.level.isK1,
+      level: options.level
```

Comment:
Good point. The options in question are pretty app-specific (`.availableBlocks`, `.ifPlaceOptions`, `.placeBlockOptions`), so may want to put those in to a separate `options.level.blockOptions` object and pass that through. (Will need to work out where in the initialization process the app can do the .level.* -> .level.blockOptions.* translation before appMain gets called, though...)